make/php-geoip.mk

```diff
@@ -0,0 +1,230 @@
+#
+# php-geoip
+#
+###########################################################
+#
+# PHP_GEOIP_VERSION, PHP_GEOIP_SITE and PHP_GEOIP_SOURCE define
+# the upstream location of the source code for the package.
+# PHP_GEOIP_DIR is the directory which is created when the source
+# archive is unpacked.
+# PHP_GEOIP_UNZIP is the command used to unzip the source.
+# It is usually "zcat" (for .gz) or "bzcat" (for .bz2)
+#
+# You should change all these variables to suit your package.
+# Please make sure that you add a description, and that you
+# list all your packages' dependencies, seperated by commas.
+#
+# If you list yourself as MAINTAINER, please give a valid email
+# address, and indicate your irc nick if it cannot be easily deduced
+# from your name or email address.  If you leave MAINTAINER set to
+# "NSLU2 Linux" other developers will feel free to edit.
+#
+PHP_GEOIP_SITE=https://pecl.php.net/get
+PHP_GEOIP_VERSION=1.1.1
+PHP_GEOIP_SOURCE=geoip-$(PHP_GEOIP_VERSION).tgz
+PHP_GEOIP_DIR=geoip-$(PHP_GEOIP_VERSION)
+PHP_GEOIP_UNZIP=zcat
+PHP_GEOIP_MAINTAINER=NSLU2 Linux <nslu2-linux@yahoogroups.com>
+PHP_GEOIP_DESCRIPTION=PHP geoip extension based on libGeoIP library.
+PHP_GEOIP_SECTION=net
+PHP_GEOIP_PRIORITY=optional
+PHP_GEOIP_DEPENDS=php, geoip
+PHP_GEOIP_SUGGESTS=
+PHP_GEOIP_CONFLICTS=
+
+#
+# PHP_GEOIP_IPK_VERSION should be incremented when the ipk changes.
+#
+PHP_GEOIP_IPK_VERSION=1
+
+#
+# PHP_GEOIP_PATCHES should list any patches, in the the order in
+# which they should be applied to the source code.
+#
+#PHP_GEOIP_PATCHES=$(PHP_GEOIP_SOURCE_DIR)/configure.patch
+
+#
+# If the compilation of the package requires additional
+# compilation or linking flags, then list them here.
+#
+PHP_GEOIP_CPPFLAGS=
+PHP_GEOIP_LDFLAGS=-lGeoIP
+
+#
+# PHP_GEOIP_BUILD_DIR is the directory in which the build is done.
+# PHP_GEOIP_SOURCE_DIR is the directory which holds all the
+# patches and ipkg control files.
+# PHP_GEOIP_IPK_DIR is the directory in which the ipk is built.
+# PHP_GEOIP_IPK is the name of the resulting ipk files.
+#
+# You should not change any of these variables.
+#
+PHP_GEOIP_BUILD_DIR=$(BUILD_DIR)/php-geoip
+PHP_GEOIP_SOURCE_DIR=$(SOURCE_DIR)/php-geoip
+PHP_GEOIP_IPK_DIR=$(BUILD_DIR)/php-geoip-$(PHP_GEOIP_VERSION)-ipk
+PHP_GEOIP_IPK=$(BUILD_DIR)/php-geoip_$(PHP_GEOIP_VERSION)-$(PHP_GEOIP_IPK_VERSION)_$(TARGET_ARCH).ipk
+
+.PHONY: php-geoip-source php-geoip-unpack php-geoip php-geoip-stage php-geoip-ipk php-geoip-clean php-geoip-dirclean php-geoip-check
+
+#
+# This is the dependency on the source code.  If the source is missing,
+# then it will be fetched from the site using wget.
+# $(PHP_GEOIP_URL) holds the link to the source,
+# which is saved to $(DL_DIR)/$(PHP_GEOIP_SOURCE).
+# When adding new package, remember to place sha512sum of the source to
+# scripts/checksums/$(PHP_GEOIP_SOURCE).sha512
+#
+$(DL_DIR)/$(PHP_GEOIP_SOURCE):
+	$(WGET) -P $(DL_DIR) $(PHP_GEOIP_SITE)/$(PHP_GEOIP_SOURCE)
```

Comment:
This should be changed to:
```
	$(WGET) -P $(@D) $(PHP_GEOIP_SITE)/$(@F) || \
	$(WGET) -P $(@D) $(SOURCES_NLO_SITE)/$(@F)
```
This way the recipe looks "cleaner", and we add rule to fetch from `$(SOURCES_NLO_SITE)` mirror if upstream isn't available (http://ipkg.nslu2-linux.org/sources)

Reply:
ok; done